source/guide_libreoffice.rst

```diff
@@ -0,0 +1,444 @@
+    web backends handle TLS.
+  * ``DONT_GEN_SSL_CERT`` must be set so that LibreOffice does not generate its
+    own certificate.
+  * The option ``-v <host path>:<guest path>`` maps a path in the host file
+    system to a path in the container. ``map_start_collabora`` is a hack
+    specific to the docker image `collabora/code`_, which otherwise doesn't
+    find its internal startup script.
+  * ``--user=101`` lets udocker emulate the correct user ID in the container.
+
+The script must be executable and should only be readable by your user, because
+it contains the admin password:
+
+::
+
+  [isabell@stardust ~]$ chmod 0700 $HOME/etc/libreoffice/run
+  [isabell@stardust ~]$
+
+
+Configure webserver
+-------------------
+
+The LibreOffice web server listens on port 9980 by default. You can find out if
+another service is already listening on this port by running ``netstat -tulpen
+| grep 9980``. If nothing is displayed, everything is fine. If the port is
+blocked, choose another port ``<port>`` for LibreOffice by appending the flag
+``--port=<port>`` to ``extra_params`` in ``$HOME/etc/libreoffice/run``.
+
+.. include:: includes/web-backend.rst
+
+If you setup the web backend to a subpath of the document root, you must also
+assign it to the option ``net.service_root`` in ``extra_params`` (with
+``--o:``) or in ``$HOME/etc/libreoffice/loolwsd.xml``.
+
+Last, if you want to embed LibreOffice in a website served from another
+domain or port than ``isabell.uber.space:443``, such as a Nextcloud,
+you must change the :manual:`web headers <web-headers>`. This is necessary
+because Uberspace sets ``X-Frame-Options: SAMEORIGIN`` by default, which allows
+embedding only in websites served from the same domain and port.
+
+::
+
+  [isabell@stardust ~]$ uberspace web header suppress isabell.uber.space X-Frame-Options
+  [isabell@stardust ~]$
+
+
+Setup daemon
+------------
+
+Create a service configuration file ``$HOME/etc/services.d/libreoffice.ini``
+
+.. code-block:: ini
+
+  [program:libreoffice]
+  command=%(ENV_HOME)s/etc/libreoffice/run
+  startsecs=45
+  autorestart=yes
+  stopasgroup=yes
+  killasgroup=yes
+
+.. include:: includes/supervisord.rst
+
+Test and use the service
+========================
+
+Run on the command line:
+
+::
+
+  [isabell@stardust ~]$ curl -k https://isabell.uber.space
+  OK
+  [isabell@stardust ~]$
+
+If you get an ``OK`` the service is running and can be accessed.
+
+Now you can navigate to https://isabell.uber.space/loleaflet/dist/admin/admin.html
+to try out the admin console and configure your file host at
+``cloud.example.org`` to use your libreoffice installation.
+
+If you configured local file system storage, you can edit files *accessible in
+the container* under the URL
+https://isabell.uber.space/loleaflet/dist/loleaflet.html?file_path=file:///PATH/TO_DOC.
+This is mainly intended for development and described in more detail in the
```

Comment:
Please remove all mentions of the local mode, as it's intended for development only.